client/client.go

```diff
@@ -191,8 +191,11 @@ func (cli *Client) Close() error {
 
 // getAPIPath returns the versioned request path to call the api.
 // It appends the query parameters to the path if they are not empty.
-func (cli *Client) getAPIPath(p string, query url.Values) string {
+func (cli *Client) getAPIPath(p string, query url.Values) (string, error) {
 	var apiPath string
+	if strings.Contains(p, "../") {
```

Comment:
I was discussing earlier with @vdemeester if there are other potentially dangerous variations. If we could detect something like `if value !== escapedValue(value)`